### PR TITLE
[Python-SDK] Fixup extension methods

### PR DESCRIPTION
**Pull request by bbonenfant**
[INT-980]

[INT-980]: https://pachyderm.atlassian.net/browse/INT-980?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: python-sdk/pachyderm_sdk/api/worker/extension.py [hunk 1]
```diff
@@ -49,5 +49,4 @@ def batch_datum(self) -> ContextManager:
             yield
         except Exception as error:
             self.__error = repr(error)
-            # TODO: Probably want better logging here than a print statement.
```
Comment (bbonenfant) [inline python-sdk/pachyderm_sdk/api/worker/extension.py:52]: Will be addressed by INT-989

File: python-sdk/tests/test_pps.py [hunk 2]
```diff
@@ -108,7 +108,6 @@ class TestUnitDatums:
 
     @staticmethod
     def test_datums(client: TestClient, default_project: bool):
-        # TODO: Why is this test sometimes slow?
```
Comment (bbonenfant) [inline python-sdk/tests/test_pps.py:111]: Will be addressed by [INT-1040]

File: python-sdk/tests/fixtures.py [hunk 3]
```diff
@@ -127,7 +124,7 @@ def _generate_name(self) -> str:
                 .replace("/", "-")
                 .replace(":", "-")
                 .replace(".py", "")
-        )[:40]  # TODO: Make this the maximum it can be.
+        )[:40]
```
Comment (chainlink) [inline python-sdk/tests/fixtures.py:127]: Nit: add a comment as to where this magic number comes from on the core pach side

File: python-sdk/pachyderm_sdk/api/pfs/extension.py [hunk 4]
```diff
@@ -231,7 +255,7 @@ def put_file_from_file(
         self._stub.put_file_from_file(commit=self, path=path, file=file, append=append)
         return File(commit=self._commit, path=path)
 
-    def copy_file(self, *, src: "File", dst: str, append: bool = True) -> "File":
+    def copy_file(self, *, src: "File", dst: str, append: bool = False) -> "File":
```
Comment (chainlink) [inline python-sdk/pachyderm_sdk/api/pfs/extension.py:258]: Strange that the default for copy was append

Comment (bbonenfant) [reply python-sdk/pachyderm_sdk/api/pfs/extension.py:258]: This was a mistake when moving this from python_pachyderm. It's False there, I'm not sure how these got flipped.

File: python-sdk/tests/fixtures.py (hunk 3 above)
Comment (chainlink) [reply python-sdk/tests/fixtures.py:127]: @bbonenfant I think I saw a thread about this in slack. Can you add the note here?

**Review by chainlink — APPROVED** [review]
Pre-approving pending the one last comment todo.
Next actions:
- PFS Review of byte level manipulation (+ adding tests to verify the bytes?)
- Follow on Jira Tickets

**Pull request merged**